lib/cli/src/commands/run/wasi.rs

```diff
@@ -491,8 +492,14 @@ impl Wasi {
 
                 MappedDirectory {
                     host: current_dir,
-                    guest: MAPPED_CURRENT_DIR_DEFAULT_PATH.to_string(),
+                    guest: if is_wasix {
+                        MAPPED_CURRENT_DIR_DEFAULT_PATH.to_string()
+                    } else {
+                        "/".to_string()
```

Comment:
I'd love a const about `"/"`, `const ROOT_PATH: &str = "/";` or similar. What do you think?

Reply:
I'd happily approve of someone else doing that XD `/` is well-known enough to warrant not giving it a const of its own though.